### PR TITLE
Small change to test code review functionality

### PR DESCRIPTION
**Pull request by sa-pradhan**

### CODE REVIEW

File: Season4/TeamCode/src/main/java/org/firstinspires/ftc/team13180s3/MoveServo.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public void execute(){
         if(time.milliseconds() >= TimeoutMs){
             state = State.STOP;
         }
-        //every five seconds move the servo
+        //every five seconds move the servo and change the position
         else if(time.milliseconds() >= 15000){
             servo.setPosition(1);
```
Comment (sa-pradhan) [inline Season4/TeamCode/src/main/java/org/firstinspires/ftc/team13180s3/MoveServo.java:35]: sample code review comment: servo should be set to position 0 rather position 1.

File: Season4/TeamCode/src/main/java/org/firstinspires/ftc/team13180s3/MoveServo.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public void execute(){
         if(time.milliseconds() >= TimeoutMs){
             state = State.STOP;
```
Comment (sa-pradhan) [inline Season4/TeamCode/src/main/java/org/firstinspires/ftc/team13180s3/MoveServo.java:31]: another sample code review comment: Is State.STOP the right state here?

**Review by pratyushseth — APPROVED** [review]
Looks like a noop.

**Comment by pratyushseth** [comment]
Finishing the review functionality test which Salil started a while ago.

**Pull request closed without merging**